src/grammar.pegjs

```diff
@@ -2338,33 +2338,35 @@ tornado_comment
   }
 
 tornado_body
-  = start:tornado_body_tag_start ws* rbrace h:html bodies:tornado_bodies end:tornado_body_tag_end &{
+  = start:tornado_body_tag_start ws* rbrace contents:nodes bodies:tornado_bodies end:tornado_body_tag_end &{
     if(!end || start.key !== end.key) {
       error('Expected end tag for "' + start.key + '" ' + start.type + ' body, start tag was "' + start.key + '" and end tag was "' + end.key + '"');
     }
     return true;
   }
   {
+    // combine the default body into bodies
     start.bodies = bodies;
-    start.body = h;
+    start.bodies.unshift(['TORNADO_BODY', {name: null, type: 'bodies', body: contents}]);
```

Comment:
Can we give the default body a name (maybe `default` or `main`) so that it can be more easily referenced in helpers.